controllers/component_build_controller.go

```diff
@@ -73,8 +73,7 @@ const (
 )
 
 type BuildStatus struct {
-	Simple *SimpleBuildStatus `json:"simple,omitempty"`
-	PaC    *PaCBuildStatus    `json:"pac,omitempty"`
+	PaC *PaCBuildStatus `json:"pac,omitempty"`
```

Comment:
Since there is currently only one way to build, can we merge `PaCBuildStatus` into `BuildStatus`?

Reply:
no I don't want to do that, we can have new status in the future

also not sure if that wouldn't require some migration